docs/self-managed/connectors-deployment/connectors-configuration.md

```diff
@@ -93,6 +93,26 @@ CONNECTOR_HTTPJSON_TYPE=non-default-httpjson-task-type
 
 Providing secrets to the runtime environment can be achieved in different ways, depending on your setup.
 
+### Default secret provider
+
+:::caution
+By default, all environment variables can be used as connector secrets.
+:::
+
+To limit the environment that can be accessed by the default secret provider, please configure a prefix, for example:
+
+```bash
+export CAMUNDA_CONNECTOR_SECRETPROVIDER_ENVIRONMENT_PREFIX='SECRETS_'
```

Comment:
Lets use a different example prefix here as this can confuse users as one could think that this prefix is related to our `{{secrets.}}` prefix. Something like `my_secrets_prefix` or similar?